NEWS.md

```diff
@@ -161,6 +161,7 @@ The `teal` package contains the code to create apps (`teal::init`), to create a
 * Updated the "filter panel collapse" icon to remove warnings when using `shiny` version >= 1.7.
 * Removed some of the non-exported, debugging modules.
 * Updated the footer tag style to be less visible.
+* Updated the header tag style to enhance element containment.
```

Comment:
I think this News entry should be in a current version not former

Reply:
my bad! updating this.